src/components/Browse/Genomes/CataloguesList/index.tsx

```diff
@@ -26,25 +26,43 @@ const BrowseGenomesByCatalogue: React.FC = () => {
     page_size: pageSize,
     lineage: biome,
   });
-
   const { data: allCatalogues } = useMGnifyData('genome-catalogues', {
     page_size: 50,
   });
 
   const columns = React.useMemo(
     () => [
+      {
+        id: 'catalogue-biome-label',
+        Header: 'Catalogue biome',
+        accessor: 'attributes.catalogue-biome-label',
+        aggregate: 'count',
+        Aggregated: ({ value }) => `${value} catalogues`,
+      },
       {
         id: 'biome',
-        Header: 'Biome',
+        Header: '',
         accessor: (catalogue) => catalogue.relationships.biome.data?.id,
         Cell: ({ cell }) => (
           <span
             className={`biome_icon icon_xs ${getBiomeIcon(cell.value)}`}
             style={{ float: 'initial' }}
           />
         ),
-        disableSortBy: true,
+        aggregate: (biomes) => biomes[0],
         className: 'mg-biome',
+        disableSortBy: true,
+        disableGroupBy: true,
+      },
+      {
+        id: 'catalogue_type',
+        Header: 'Type',
+        accessor: 'attributes.catalogue-type',
+        Cell: ({ cell }) => cell.value[0].toUpperCase() + cell.value.slice(1),
```

Comment:
Hi @SandyRogers when I pulled this locally, I had issues with cell.value possibly being undefined. 
The following adjustments addressed it:

`
        Cell: ({ cell }) => {
          return cell.value
            ? cell.value[0].toUpperCase() + cell.value.slice(1)
            : '';
        },

        aggregate: (catTypes) => {
          const validItems = catTypes
            .filter(Boolean)
            .map((catType) => `${catType.slice(0, 4)}.`);

          return validItems.length ? validItems.join(', ') : '';
        },

`

Reply:
Is that because you weren't using the corresponding API branch, perhaps? Once the API is deployed there should never be null catalogue types.

Reply:
I was fetching data from https://wwwint.ebi.ac.uk/metagenomics/api/v1/
I also tried using the latest master branch of the API